generators/server/templates/_pom.xml

```diff
@@ -974,9 +977,6 @@
                     Enable the line below to have remote debugging of your application on port 5005
                     <jvmArguments>-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=5005</jvmArguments>
                     -->
-                    <arguments>
```

Comment:
Seems like same behavior when we package the war as well using prod profile.SO the issue might be there even without this PR for war files. so somehow the profile set in `application.yml` is not used 100%, atleast its not used for loading DB, might be related to https://github.com/jhipster/generator-jhipster/issues/3618